share/tools/web_config/js/colorutils.js

```diff
@@ -393,19 +393,32 @@ function master_color_for_color(color_str) {
 
 /* Given a color name, like 'normal' or 'red' or 'FF00F0', return an RGB color string (or empty string) */
 function interpret_color(str) {
-	str = str.toLowerCase()
-	if (str == 'black') return '000000'
-	if (str == 'red') return 'FF0000'
-	if (str == 'green') return '00FF00'
-	if (str == 'brown') return '725000'
-	if (str == 'yellow') return 'FFFF00'
-	if (str == 'blue') return '0000FF'
-	if (str == 'magenta') return 'FF00FF'
-	if (str == 'purple') return 'FF00FF'
-	if (str == 'cyan') return '00FFFF'
-	if (str == 'white') return 'FFFFFF'
-	if (str == 'normal') return ''
-	return str
+    str = str.toLowerCase();
+    if (str == 'black') return '000000';
+    if (str == 'red') return '800000';
```

Comment:
Huh, those few colors in the bright column for X are surprising (in a good way.) They're actually quite near what I end up using in my terminal configurations.